### PR TITLE
Reorder configuration

### PR DESCRIPTION
**Pull request by ludeeus**
https://www.figma.com/file/RSUZAyEXLdvmPWhonsI2xM/Menu?node-id=176%3A20671

### CODE REVIEW

**Comment by bramkragten** [comment]
What needs to be done for this still? Should we wait on it for a release or not?

File: src/panels/config/dashboard/ha-config-navigation.ts [hunk 1]
```diff
@@ -19,10 +20,16 @@ class HaConfigNavigation extends LitElement {
...
   @property() public pages!: PageNavigation[];
 
+  @property() public externalConfig?: ExternalConfig;
+
   protected render(): TemplateResult {
     return html`
       ${this.pages.map((page) =>
-        canShowPage(this.hass, page)
+        (
+          page.path === "#external-app-configuration"
```
Comment (bramkragten) [inline src/panels/config/dashboard/ha-config-navigation.ts:29]: ?
```suggestion
          page.component === "mobile_app"
```

File: src/panels/config/dashboard/ha-config-navigation.ts [hunk 2]
```diff
@@ -77,6 +84,16 @@ class HaConfigNavigation extends LitElement {
 
   private _entryClicked(ev) {
     ev.currentTarget.blur();
+    if (
+      ev.currentTarget.parentElement.href.endsWith(
+        "#external-app-configuration"
+      )
```
Comment (bramkragten) [inline src/panels/config/dashboard/ha-config-navigation.ts:90]: Not sure I like this, the url will stay changed, and it adds a entry to the history...

File: src/panels/config/dashboard/ha-config-navigation.ts [hunk 3]
```diff
@@ -77,6 +84,16 @@ class HaConfigNavigation extends LitElement {
 
   private _entryClicked(ev) {
     ev.currentTarget.blur();
+    if (
+      ev.currentTarget.parentElement.href.endsWith(
```
Comment (bramkragten) [inline src/panels/config/dashboard/ha-config-navigation.ts:88]:
```suggestion
      ev.currentTarget.component === "mobile_app"
```

File: src/panels/config/dashboard/ha-config-navigation.ts (hunk 1 above)
Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-navigation.ts:29]: No, your instance have the integration loaded, but this should only show if you are visiting from a app

File: src/panels/config/dashboard/ha-config-navigation.ts (hunk 2 above)
Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-navigation.ts:90]: This is how it used to be, but in `ha-sidebar`

File: src/panels/config/dashboard/ha-config-navigation.ts (hunk 3 above)
Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-navigation.ts:88]: No, see <https://github.com/home-assistant/frontend/pull/10817#discussion_r763467507>

File: src/panels/config/dashboard/ha-config-navigation.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/config/dashboard/ha-config-navigation.ts:29]: yes, this still does that?

**Review by balloob — APPROVED** [review]

**Pull request merged**